python/lib/services.py

```diff
@@ -0,0 +1,3 @@
+from statsd import StatsClient
+
+statsd = StatsClient(prefix="govuk.app.content-tagger")
```

Comment:
I think something like `stats.gauges.govuk.topic-taxonomy` might be better?

Reply:
Or rather, something sensible within that namespace.

Reply:
ah yes. Bit too much cut-and-paste. I'll change